include/llama.h

```diff
@@ -467,10 +467,14 @@ extern "C" {
     LLAMA_API uint32_t llama_n_ubatch   (const struct llama_context * ctx);
     LLAMA_API uint32_t llama_n_seq_max  (const struct llama_context * ctx);
 
-    LLAMA_API enum llama_pooling_type llama_pooling_type(const struct llama_context * ctx);
-
-    LLAMA_API enum llama_vocab_type   llama_vocab_type  (const struct llama_model * model);
-    LLAMA_API enum llama_rope_type    llama_rope_type   (const struct llama_model * model);
+    LLAMA_API enum llama_pooling_type llama_get_pooling_type(const struct llama_context * ctx);
+    LLAMA_API enum llama_vocab_type   llama_get_vocab_type  (const struct llama_model * model);
+    LLAMA_API enum llama_rope_type    llama_get_rope_type   (const struct llama_model * model);
+
+    // DEPRECATED: use the API above
+    //LLAMA_API enum llama_pooling_type llama_pooling_type(const struct llama_context * ctx);
+    //LLAMA_API enum llama_vocab_type   llama_vocab_type  (const struct llama_model * model);
+    //LLAMA_API enum llama_rope_type    llama_rope_type   (const struct llama_model * model);
```

Comment:
Deprecated means that the functions are still available and will be removed soon. This is a breaking API change.